docs/sources/datasources/influxdb/_index.md

```diff
@@ -32,7 +32,7 @@ Name        | Description
 `Name`      | The data source name. This is how you refer to the data source in panels and queries. We recommend something like `InfluxDB-InfluxQL`.
 `Default`   | Default data source means that it will be pre-selected for new panels.
 `URL`       | The HTTP protocol, IP address and port of your InfluxDB API. InfluxDB API port is by default 8086.
-`Access`    | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser.
+`Access`    | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser (**WARNING**: Browser access is deprecated and will be removed in a future release).
```

Comment:
```suggestion
`Access`    | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser.
**Note**: Browser access is deprecated and will be removed in a future release.
```

Reply:
It would be better to add this as a note, instead of a warning.

Reply:
@achatterjee-grafana thanks, applied!